api/src/accounts.rs

```diff
@@ -185,7 +185,7 @@ pub struct Account {
     /// Address of account
     address: Address,
     /// Lookup ledger version
-    ledger_version: u64,
+    pub requested_version: u64,
```

Comment:
I think there is other code in the api directory where we use `ledger_version` as the var name. Not a huge deal but it might be confusing if we use different names, especially since the query param is called `ledger_version`.

Reply:
Changed back.